CMakeLists.txt

```diff
@@ -1,47 +1,19 @@
 cmake_minimum_required(VERSION 3.15)
 project(KickCAT)
 
+option(BUILD_MASTER "Build master library" ON)
+
+option(BUILD_SLAVE        "Build slave library" OFF)
```

Comment:
Note to future selves, we will have to add these option to the conan recipes (here: https://github.com/conan-io/conan-center-index/blob/d57266c3ad9eb3dbda6e369df33a0daa764a39c3/recipes/kickcat/all/conanfile.py#L51) for it to build te lib correctly. I know by default the slave stack will always be disabled but it is always nice to have all option info there.

Reply:
Theses options are on the way to be removed: the whole package will be built.